examples/sysl/endpoints.wbnf

```diff
@@ -0,0 +1,18 @@
+// -------------- Endpoint     --------------- //
```

Comment:
remove these headers if we are splitting up the grammar into multiple files

Reply:
removed